main/dovecot/APKBUILD

```diff
@@ -3,11 +3,11 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dovecot
-pkgver=2.2.27
+pkgver=2.2.29.1
 pkgrel=2
-_pigeonholever=0.4.16
+_pigeonholever=0.4.18
 _pluginextdataver=39
-_pkgvermajor=${pkgver%.*}
+_pkgvermajor=$(echo $pkgver | cut -d. -f1-2)
```

Comment:
I can't think of any easy way of dealing with changing number of dots (i.e. that would cope with 2.2.29.1 and 2.2.30) without a subshell . Either we have a subshell here so we don't have to change with it in the future, or I change the pattern for this current case only.

Which would you rather?